### ISSUE TITLE
dataset questions

### ISSUE THREAD

**Issue by miaoct** [JStehouwer/FFD_CVPR2020#10]
1、	Whether the published dataset ( FFHQ、FaceAPP、StarGAN、PGGAN、StyleGAN ) has been randomly selected ? And How to generate starGAN mask, how to determine the specific CelebA picture used ?
2、	I have downloaded the FF++、CelebA and DeepFaceLab dataset, how to randomly select the training set, test set and verification set ? And how to set the random seed ?
3、	Which data sets need align processing, and how, please specify ?

Thank you for your work, it is very good, I will follow your work, but now the problem of dataset makes my work difficult, I hope to get your help.

**Comment by multimedia-infomation-security** [comment]
have you solved your problem?

**Comment by Lime-song** [comment]
have you solved your problem?

**Comment by zhipeixu** [comment]
have you solved your problem?
